index.html

```diff
@@ -25,13 +26,153 @@ <h1>I’m Patrick
 			you see and have a project you need coded, don’t
 		  hestiate to contact me.</p>
 		<a href="">LET’S CONNECT</a>
-		<ul>
+		<ul class="tags">
 			<li><a href=""><img src="/img/github.svg" alt=""></a></li>
 			<li><a href=""><img src="/img/Linkedin.svg" alt=""></a></li>
 			<li><a href=""><img src="/img/medium.svg" alt=""></a></li>
 			<li><a href=""><img src="/img/twiter.svg" alt=""></a></li>
 			<li><a href=""><img src="img/chill.svg" alt=""></a></li>
 		</ul>
 	</section>
+
+	<section class="work">
+		<div>
+			<img src="/img/tonic1.svg" alt="">
+			<h2>TONIC</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
+		</div>
+
+		<div>
+			<img src="/img/multistories1.svg" alt="">
+			<h2>Multi-Post
+				Stories</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
+		</div>
+
+		<div>
+			<img src="/img/tonic2.svg" alt="">
+			<h2>TONIC</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
+		</div>
+
+		<div>
+			<img src="/img/multistories2.svg" alt="">
+			<h2>Multi-Post
+				Stories</h2>
+			<ul class="work-exp">
+				<li>Canopy</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>Back End Dev</li>
+				<li><img src="/img/circle.svg" alt=""></li>
+				<li>2015</li>
+			</ul>
+			<p>A daily selection of privately personalized reads; 
+				no accounts or sign-ups required.</p>
+      <ul class="lang">
+				<li><a href="">html</a></li>
+				<li><a href="">css</a></li>
+				<li><a href="">javascript</a></li>
+			</ul>
+
+			<button class="btn">See Project</button>
```

Comment:
This is a nice button that you created here 💯 but in order to match the Figma a little bit more and to make your project cooler 😎 The button should follow the button styling: Active, Hover and Pressed [see design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=WRpq1TpJZoIQMeK1-0). Take a look below:

![Screenshot 2023-01-11 at 20 26 03](https://user-images.githubusercontent.com/45739963/211953961-7953ce3e-592f-4441-be86-f77d0efb2d9b.png)